hknweb/candidate/views.py

```diff
@@ -28,11 +30,14 @@ def get_context_data(self):
         announcements = Announcement.objects \
                 .filter(visible=True) \
                 .order_by('-release_date')
+        today = datetime.date.today()
+        upcoming_events = Event.objects.filter(start_time__range=(today, today + datetime.timedelta(days=7))).order_by('start_time')
         context = {
             'num_pending' : challenges.filter(reviewed=False).count(),
             'num_rejected' : reviewed_challenges.filter(confirmed=False).count(),
             'num_confirmed' : reviewed_challenges.filter(confirmed=True).count(),
-            'announcements' : announcements
+            'announcements' : announcements,
+            'upcoming_events': upcoming_events
```

Comment:
I tend to put trailing commas to minimize future diffs:

‘upcoming_events’: upcoming_events,

Reply:
Fixed